rpcserver/macaroons.go

```diff
@@ -0,0 +1,12 @@
+package rpcserver
+
+import "gopkg.in/macaroon-bakery.v2/bakery"
+
+// RPCServerPermissions is the set of macaroon permissions needed for each
+// api.
+var RPCServerPermissions = map[string][]bakery.Op{
+	"/offersrpc.Offers/SendOnionMessage": {{
```

Comment:
Do we want to have macaroons? How will we bake an external subserver mac + use in lnd itests?

Reply:
Works in itests. 

Will need to bake our own to use (or use subset of perms that admin mac already has)